tests/framework/extensions/rke1/nodepools/nodepools.go

```diff
@@ -57,61 +64,78 @@ func NodePoolSetup(client *rancher.Client, nodeRoles []NodeRoles, ClusterID, Nod
 	return &nodePoolConfig, nil
 }
 
-// ScaleWorkerNodePool is a helper method that will add a worker node pool to the existing RKE1 cluster. Once done, it will scale
-// the worker node pool to add a worker node, scale it back down to remove the worker node, and then delete the worker node pool.
-func ScaleWorkerNodePool(client *rancher.Client, nodeRoles []NodeRoles, ClusterID, NodeTemplateID string) error {
-	nodePoolConfig := management.NodePool{
-		ClusterID:               ClusterID,
-		ControlPlane:            false,
-		DeleteNotReadyAfterSecs: 0,
-		Etcd:                    false,
-		HostnamePrefix:          "auto-rke1-scale-" + ClusterID,
-		NodeTemplateID:          NodeTemplateID,
-		Quantity:                1,
-		Worker:                  true,
-	}
-
-	logrus.Infof("Creating new worker node pool...")
-	nodePool, err := client.Management.NodePool.Create(&nodePoolConfig)
+// AllNodeList is a helper method that will return each node in the cluster.
+func AllNodeList(client *rancher.Client, cluster *management.Cluster, nodeRoles NodeRoles) (*management.Node, error) {
```

Comment:
Could this get a new name to match what the function is doing? Also this is similar to what I had in scale_replace_test.go, maybe we could consolidate